public/lessons/javascript/variables.md

```diff
@@ -106,7 +106,7 @@ Some things to try:
 
     let snack = "Apple"
 
-![snack-apple](snack-apple.svg)
+![snack-apple](/images/snack-apple.svg)
```

Comment:
@Habenzy This one may also be broken, but it's possible that the problem is Github does not know which image to use as the Bootcamp site finds the image from the top level `images` directory.
Just double check.

Reply:
This one is fine too.